src/implementations/twig/components/site-footer/site-footer.html.twig

```diff
@@ -0,0 +1,166 @@
+{% spaceless %}
+
+{#
+  Parameters:
+  - "variant" (string) (default: '') core, standardised or harmonised
```

Comment:
do we still need this "variant" parameter (and corresponding css classes)?

Reply:
I actually wanted to see this point with you, a good approach would be a “negative” boolean param?
instead of having the `—core`, `—standradised` or even `—harmonised` classes, we would just have a `—negative` class. this could also apply for the new **page-header** (https://github.com/ec-europa/europa-component-library/pull/2337) as well as the **breadcrumb**.

Reply:
for now I added a param `split_columns` that generates a `--split-columns` class modifier on used for core EC